Tools/WebKitTestRunner/Configurations/WebKitTestRunner-internal.entitlements

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>com.apple.WebKitTestRunner</string>
+	</array>
+	<key>com.apple.private.webkit.adattributiond.testing</key>
+	<true/>
+	<key>com.apple.security.temporary-exception.sbpl</key>
```

Comment:
Duplicating the entitlements is of course dangerous. If producing them with a script like we do in WebKit itself is too much hassle, perhaps the opensource version should be renamed to `WebKitTestRunner-opensource.entitlements`, as a prompt that it's not the only file to modify?

Reply:
I don't think it's high-risk for slipping out of sync, as any Apple engineer will need to update the internal entitlements for their at-desk builds, and the open source entitlements for any added tests to pass EWS.

re: naming, the `-internal` suffix is precedented by `Tools/TestWebKitAPI/Configurations/TestWebKitAPI-macOS-internal.entitlements`. I'd prefer matching that convention, but if you really want to change it we should change it in both places.

Reply:
I think that it's better for both to have suffixes (`-internal` and `-opensource`), as having no suffix for one of them is confusing. It confused me when discussing this PR.

Would be even better to generate them.

This is not a blocker for landing this PR.